### ISSUE TITLE
Field "name" of type ServiceEventName in ServiceEventCharging has invalid value 'charging-status-changed' 

### ISSUE THREAD

**Issue by prvakt** [skodaconnect/myskoda#158]
Hi, from time to time I see following error when testing myskoda[cli]
```
2024-11-06 08:24:40 ubuntu myskoda.mqtt[2631] DEBUG Received PUBLISH (d0, q0, r0, m0), 'xxxxxxxx-xxxx-xxxx-xxxx-xxxxxxxxxxxx/TMBXXXXXXXXXXXXXX/service-event/charging', ...  (237 bytes)
2024-11-06 08:24:40 ubuntu myskoda.mqtt[2631] DEBUG Message (service-event) received for TMBXXXXXXXXXXXXXX on topic charging: b'{"version":1,"traceId":"xxxxxxxx-xxxx-xxxx-xxxx-xxxxxxxxxxxx","timestamp":"2024-11-06T07:24:40Z","producer":"SKODA_MHUB","name":"charging-status-changed"," data":{"userId":"xxxxxxxx-xxxx-xxxx-xxxx-xxxxxxxxxxxx","vin":"TMBXXXXXXXXXXXXXX"}}'
2024-11-06 08:24:40 ubuntu myskoda.mqtt[2631] WARNING Exception parsing MQTT event: Field "name" of type ServiceEventName in ServiceEventCharging has invalid value 'charging-status-changed' 
```

**Comment by WebSpider** [comment]
Duplicate of #148

**Comment by WebSpider** [comment]
This has been solved in v1.7.2 of the integration and v0.8.9 of the python library
